CONTRIBUTE.md

```diff
@@ -26,3 +26,21 @@ This is a community built document. Dive in and help out!
         ```sh
         (relevant branch)$ git blah blah
         ```
+
+## Translation
+
+We are using with a translation tool called [GitLocalize][gl] and follow the steps to get started with your contribution:
```

Comment:
We use a translation called GitLocalize. Please follow these steps to get started with your contributions: